src/integration.d.ts

```diff
@@ -13,7 +13,7 @@ export interface IntegrationStatic<Settings extends object, Options extends obje
    * Creates a new integration.
    * @param settings The integration settings
    */
-  new (settings: Settings): Integration<Settings, Options>
+  new (settings: Settings, features: string[]): Integration<Settings, Options>
```

Comment:
Should this parameter have a default value?

Reply:
The default is handled on [this line](https://github.com/segmentio/integration/pull/91/files/ba184815ea1c260c035d105c21e568282b7b7997#diff-0e0c28a0622c45ce008fe018cf0601f2R48) so we can also fall back to an empty array on `null` values. (iirc, default parameters only applies to `undefined` values)

Reply:
I don't think so since this is a type def